src/components/Drawer.js

```diff
@@ -163,7 +163,9 @@ const Drawer = React.createClass({
                     onClick={this.close}
                     primaryColor={this.props.buttonPrimaryColor}
                     type={'base'}
-                  />
+                  >
+                    <span style={styles.visuallyHidden}>Close Drawer</span>
```

Comment:
Looks like this should also have `className='visuallyHidden'` because that's how you're looking for it in Button.js.